jcloud/flow.py

```diff
@@ -136,6 +136,67 @@ async def _deploy(self):
                     logger.debug('POST /flows retry failed too...')
                     raise e
 
+    async def update(self):
+        async def _update():
+            for i in range(2):
+                try:
+                    async with aiohttp.ClientSession() as session:
+                        api_url = FLOWS_API+ "/" + self.flow_id
+                        post_params = await self._get_post_params()
+
+                        async with session.put(
+                            url=api_url,
+                            headers=self.auth_header,
+                            **post_params,
+                        ) as response:
+                            json_response = await response.json()
+                            _exit_if_response_error(
+                                response,
+                                expected_status=HTTPStatus.ACCEPTED,
+                                json_response=json_response,
+                            )
+
+                            if self.flow_id is not json_response['id']:
```

Comment:
How can this case happen? Just wondering

Reply:
It just hit my mind on the day one, while working with this repo. I have added add TODO to get a confirmation. So, I think is not needed, so I will delete this and update PR.

Reply:
sure